website/content/vagrant-cloud/hcp-vagrant/migration-guide.mdx

```diff
@@ -0,0 +1,149 @@
+---
+layout: vagrant-cloud
+page_title: Migrating to HCP Vagrant
+description: "Step by step guide to using Vagrant Cloud's HCP Vagrant Migration Tools"
+---
+
+# Migrating to HCP Vagrant
+
+~> Note: Vagrant Cloud will be migrated to HCP Vagrant at the end of July 2024.
+
+## Overview
+
+HCP [Vagrant Registry](https://portal.cloud.hashicorp.com/vagrant/discover), like Vagrant Cloud, is a public, searchable index of Vagrant boxes that allows box owners to host and share artifacts.
+
+To take advantage of shared infrastructure and platform investments available on the Hashicorp Cloud Platform (HCP), Hashicorp is migrating all existing boxes to HCP Vagrant Registry and retiring the current Vagrant Cloud rails app at the end of July 2024.
+
+Migrated organizations will be permanently accessible at their original Vagrant Cloud URLs and won’t require changes to user workflows. Migrated registries will have access to the modern HCP Vagrant UI, an improved search experience, and free private boxes.
+
+Those who take no action will have their boxes migrated as part of the site wide migration. The Vagrant team will maintain redirects to ensure users can expect all existing URLs and workflows to operate as usual after migration regardless of how boxes are migrated.
+
+## Prerequisites
+
+Before you continue with this guide, you should familiarize yourself with a few new resource management concepts in HCP and HCP Vagrant Registry.
+
+###### Registries
+
+The collection that holds a user’s boxes, the registry name is incorporated into the first half of resource names, e.g `hashicorp/atlas`
+
+~> Note: An HCP registry is analogous to an Organization in Vagrant Cloud. A given HCP project may have an unlimited number of registries.
+
+##### HCP Organizations
+
+[HCP organizations](/hcp/docs/hcp/admin/orgs) are parent-level entities that can have up to ten projects.A single HCP account can be a member of multiple organizations, as an invited user, but can only create and own a single organization.
```

Comment:
```suggestion
[HCP organizations](/hcp/docs/hcp/admin/orgs) are parent-level entities that can have up to ten projects. A single HCP account can be a member of multiple organizations, as an invited user, but can only create and own a single organization.
```

Reply:
The idea with this change was to use the header-anchor hooks. I'd originally tried to create explicit anchors but for some reason the links to them just didn't jump back up the page.  It's not a long guide so maybe its fine to drop but I just wanted to explain the functionality that motivated this.